pootle/static/js/editor/app.js

```diff
@@ -577,7 +594,9 @@ PTL.editor = {
   },
 
   canNavigate() {
-    if (this.isUnitDirty) {
+    if (this.isUnitDirty ||
+        (PTL.editor.suggestionFeedbackForm &&
+         PTL.editor.suggestionFeedbackForm.state.isDirty)) {
```

Comment:
Reading from a component instance's internal state from the outside should be avoided, especially because it's not guaranteed to work across versions. Instead the component should take a callback as a prop and it should be called when the desired state change happens.